json/src/main/scala/akka/stream/alpakka/json/JsonParsing.scala

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright (C) 2016-2018 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package akka.stream.alpakka.json
```

Comment:
I was wondering if this needs a seperate `javadsl`/`scaladsl`, since the parameters will be the same. Or is this to keep binaryCompat in the future? If so, then yeah, that'd make sense.

Reply:
Yes, the API will evolve and then Java and Scala might not use the same.